### DISCUSSION TITLE
21-06-10 일일회고

### DISCUSSION THREAD

**Discussion by GleamingStar** [d-h-k/issue-tracker#34] [회고]
이제 1주차도 하루남았습니다

**Comment by GleamingStar** [comment]
오늘은 NewIssue페이지를 완성했습니다
내일은 루크의 진행정도를 보고 SideBar따위의 공통컴포넌트를 보완할지, LabelList페이지를 만들지 정해야겠습니다
![image](https://user-images.githubusercontent.com/70461368/121493340-b5029700-ca12-11eb-8e95-76c8e1238e71.png)

**Comment by eeeesong** [comment]
오늘은 어제에 이어 애플 로그인 작업을 했습니다!

기기 키체인에 jwt 등을 저장해서 로그인 정보가 있으면 바로 이슈 창을 보여주도록 구현했습니다.

로그아웃까지 구현을 해보려 했으나, 앱에서는 바로 애플 로그아웃이 가능하지 않아서😂 로그아웃은 할수없는 앱이 됐습니다ㅎㅎㅎ (방법이 없진 않지만.. 당장은 어려울 것 같아서 우선 패스!!)

<image src=https://user-images.githubusercontent.com/72188416/121502309-01ea6b80-ca1b-11eb-8d24-6f6dbe2a765f.png width=300>

내일은 github까지 통합한 소셜로그인 부분의 구조를 가다듬고 마무리가 되면 레이블 설정 UI를 만들 것 같네요!

다들 오늘도 고생 많으셨습니다 :)

**Comment by JacksonPk** [comment]
롤로와 같이 소셜로그인 OAuth에 대해 학습 및 구현을 해보았습니다. 저는 깃헙으로 구현했는데 다행히도 jwt를 잘 받은 것 같습니다.
로그인했을 때 제 깃헙 프로필 이미지를 보여주고 싶었으나 size문제를 어떻게 해야할지 아직 파악을 하지 못했습니다.
해당 보라색이 제 이미지인데 아예 보이질 않아서 이 부분을 나중에 해결해 보려고 합니다!!
<img width="1106" alt="Screen Shot 2021-06-10 at 6 26 46 PM" src="https://user-images.githubusercontent.com/52390975/121503372-fb102880-ca1b-11eb-95fa-4ebae6e7065a.png">

그리구 내일은 제가 참여를 못하지만 못하는만큼 더 열심히 하도록 하겠습니다.
우리조 홧팅입니다!
일조 빠이아~

**Reply by eeeesong** [reply]
잭슨 이부분 제가 낼 할까여?! 아님 돌아오셔서 하실래여??

**Comment by Jiwon-JJW** [comment]
뭘 빼먹었다 했더니 10일 회고를 까먹었군요
JWT토큰을 받은 후, 라벨 전체 보기까지 구현했습니다!
토큰이 없으면 아예 접근이 불가능하게 해뒀습니다~~

**Comment by reminderclock** [comment]
뒷북좀 치겠습니다 ㅎㅎ oauth 적용해서 구현 진행중이였습니다ㅋ;;
